src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/TypesafeEnumConverter.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace SwedbankPay.Sdk.JsonSerialization

```

Comment:
Codacy found an issue: [Remove this empty namespace.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)